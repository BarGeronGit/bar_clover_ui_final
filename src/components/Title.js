import React from 'react';

const MegaTitle = {
  fontSize: '10vw',
  textShadow: '11px 1px 20px black',
  marginBottom: '0',
  justifyContent: 'center',
  flexWrap: 'nowrap',
}


export const MegaTitleClover = (props) => (
  <h1 style={MegaTitle}>{props.text}</h1>
);
