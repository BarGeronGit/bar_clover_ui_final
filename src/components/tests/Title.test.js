import React from 'react';
import {MegaTitleClover} from '../Title';
import renderer from 'react-test-renderer';

describe('Title Styled Components', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MegaTitleClover></MegaTitleClover>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })});
