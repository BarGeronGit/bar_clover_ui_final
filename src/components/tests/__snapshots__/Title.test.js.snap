// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Styled Components renders correctly 1`] = `
<h1
  style={
    Object {
      "flexWrap": "nowrap",
      "fontSize": "10vw",
      "justifyContent": "center",
      "marginBottom": "0",
      "textShadow": "11px 1px 20px black",
    }
  }
/>
`;
